package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "react-split-pane",
-  "description": "React split-pane component",
+  "name": "ubervu-react-split-pane",
+  "description": "Fork of React split-pane component",
   "main": "index.js",
-  "version": "0.1.11",
+  "version": "0.1.13",
   "repository": {
     "type": "git",
-    "url": "https://github.com/tomkp/react-split-pane"
+    "url": "https://github.com/ubervu/react-split-pane#greenkeeper-pin"
   },
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/tomkp/react-split-pane"
+    "url": "https://github.com/ubervu/react-split-pane"
   },
-  "homepage": "https://github.com/tomkp/react-split-pane",
-  "author": "tomkp <tom@tomkp.com>",
+  "homepage": "https://github.com/ubervu/react-split-pane",
+  "author": "UberVU",
```

Comment:
I found 2 forks of this, [this one](https://github.com/rohn/react-split-pane/blob/master/package.json) keeps it as it is, [this one](https://github.com/kadirahq/react-split-pane/blob/master/package.json) modifies it.
I guess nobody cares about it.